chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml

```diff
@@ -1,5 +1,32 @@
 ---
 - block:
+    - block: 
+
+      - name: Select the app pod
+        shell: >
+          kubectl get pod -l {{ a_label }} -n {{ a_ns }}
+          -o=custom-columns=:metadata.name --no-headers
+          | shuf | head -1 
+        args:
+          executable: /bin/bash
+        register: app_pod_name
+
+      - name: Record app pod name
+        set_fact:
+          app_pod: "{{ app_pod_name.stdout }}"
+
+      when: "app_pod is not defined or app_pod == ''"
```

Comment:
The same task is present on line 61-75. Can we put this task in one place only? 
Getting the node-name task is also repeated.

Reply:
Done